### PR TITLE
added steamID finder to citizens editor 

### PR DESCRIPTION
**Pull request by MasonColoretti**

### CODE REVIEW

File: container/nginx/www/js/modal.js
Comment (brianmiller) [inline container/nginx/www/js/modal.js:1]: can we be more prescriptive?  It's difficult to know that this file is used for. Maybe just move this into the main citizensEditor.php and name the classes accordingly

File: container/nginx/www/css/extra.css
Comment (brianmiller) [inline container/nginx/www/css/extra.css:1]: can we make the class names more prescriptive?  Also, why not move these to phvalheimstyles.css?

**Comment by brianmiller** [comment]
This will eventually resolve https://github.com/brianmiller/phvalheim-server/issues/29

File: container/nginx/www/admin/citizensEditor.php [hunk 1]
```diff
@@ -3,172 +3,272 @@
...
+            font-weight: bold;
+        }
+
+        .close:hover,
+        .close:focus {
+            color: black;
+            text-decoration: none;
+            cursor: pointer;
+        }
+        </style>
```
[the remark this answers is not in the document]
Comment (MasonColoretti) [reply container/nginx/www/admin/citizensEditor.php]: ok the rest has been resolved so i will do this

**Pull request merged**